modules/hikari/src/test/scala/doobie/hikari/issue/824.scala

```diff
@@ -59,12 +59,11 @@ class `824` extends munit.FunSuite {
 
     } flatMap { case (n, ds) =>
       // One final report to show that all connections are disposed
-      report(ds) *> IO((n, ds.getHikariPoolMXBean.getTotalConnections))
-
+      report(ds) *> IO.pure(n -> ds.getHikariPoolMXBean.getTotalConnections)
```

Comment:
I wouldn't recommend this change – since `ds.getHikariPoolMXBean.getTotalConnections` is impure operation, it is correct to keep it inside `IO`.